src/plots/Label/avtLabelMapper.C

```diff
@@ -466,3 +466,38 @@ avtLabelMapper::SetNodeOrigin(vtkIdType origin)
     }
 }
 
+
+// ****************************************************************************
+//  Method: avtLabelMapper::SetFullFrameScaling
+//
+//  Purpose:
+//    Sets a full frame scale factor that can be used by the mapper to
+//    compensate for the stretching that full frame mode performs on geometry.
+//
+//  Arguments:
+//    useScale : True if the scale is used.
+//    s        : The fullframe scale vector.
+//
+//  Returns:    True if any vtk mappers use the scale.
+//
+//  Note:  Taken from avtPseudocolorMapper.
+//
+//  Programmer: Alister Maguire
+//  Creation:   May 2l, 2021
```

Comment:
The date has a "2" followed by the letter "l". It should be followed by the number "1".

Reply:
Fixed!